src/Tracking/TrackingData/GivePluginSettingsData.php

```diff
@@ -0,0 +1,82 @@
+<?php
+namespace Give\Tracking\TrackingData;
+
+use Give\Tracking\Contracts\TrackData;
+use Give\Helpers\ArrayDataSet;
+use Give\Tracking\AdminSettings;
+
+/**
+ * Class GivePluginSettingsData
+ *
+ * This class represents Give plugin data.
+ *
+ * @since 2.10.0
+ * @package Give\Tracking\TrackingData
+ */
+class GivePluginSettingsData implements TrackData {
+
+	/**
+	 * Return Give plugin settings data.
+	 *
+	 * @since 2.10.0
+	 * @return array
+	 */
+	public function get() {
+		return [
+			'installDate'    => $this->getPluginInstallDate(),
+			'globalSettings' => $this->getGlobalSettings(),
+		];
+	}
+
+	/**
+	 * Returns plugin install date
+	 *
+	 * @since 2.10.0
+	 * @return int
+	 */
+	private function getPluginInstallDate() {
+		$confirmationPageID = give_get_option( 'success_page' );
+
+		return strtotime( get_post_field( 'post_date', $confirmationPageID, 'db' ) );
```

Comment:
@kjohnson You are correct. `Success Page` can not be empty otherwise donot will not able to see their donation receipt, so I think this does not require error handling.